assets/stories/skate-design/Typography.mdx

```diff
@@ -0,0 +1,126 @@
+import { Meta, Source, Unstyled } from "@storybook/blocks"
+import "../../css/storybook_reboot.scss"
+
+<Meta title="Typography" />
+
+# Global Settings / Style
+
+Examples of typography styles found in Skate, based on defaults from
+Reboot with some modifications. This roughly follows the outline of
+the <a href="https://getbootstrap.com/docs/5.3/content/typography/">
+Bootstrap typography documentation</a> adapted specifically for Skate.
+The sections that follow below are all rendered with Skate, as opposed
+to Storybook, styling.
+
+<Unstyled>
+  <p>
+    {
+      "The default font is Inter. Body text is 1rem, 400 weight, $gray-900(#141524)."
```

Comment:
suggestion(non-blocking): since we're "in the land of links", if you have time to figure it out, do you think you could try finding a way to link to this color on the colors page from this text?

These storybook docs _may_ help with figuring out linking? https://storybook.js.org/docs/writing-docs/mdx#linking-to-other-stories-and-pages

The biggest challenge I see is that the `gray` swatches don't have a targetable `id`, and neither does the `gray-900` varient. So somehow you'd need to add an id/anchor to the colors page that could be linked to? (which, actually may be easier than I'm expecting)

Reply:
Hmm, I'm looking at the color page with the inspector and not finding much in the way of useful IDs to link to particular parts on the page. Unless you have a specific idea in mind I haven't stumbled upon yet I don't think this is feasible, for now at least.

Reply:
This may be the wrong tag(but it also strangely feels like the right one?), but I think you could add an ID to use by putting an anchor above the gray `<ColorItem/> entry

```html
<a id="color-gray" />
```

Reply:
ba49be33

Reply:
Thanks!